python/ray/dag/tests/experimental/test_accelerated_dag.py

```diff
@@ -998,6 +998,26 @@ def test_composite_channel_multi_output(self, ray_start_regular_shared):
         compiled_dag.teardown()
 
 
+def test_channel_access_after_close(ray_start_regular):
```

Comment:
Use `ray_start_regular_shared` instead? It may be able to fix the CI error.

Reply:
Done, thanks!